apps/queue-processor/package.json

```
@@ -6,6 +6,7 @@
   "dependencies": {
     "@easyshell/utils": "0.1.0",
     "@easyshell/problems": "0.1.0",
+    "@easyshell/env": "0.1.0",
```

Biome (1.9.4)

[error] The key @easyshell/env was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)